packages/remix/src/build.ts

```diff
@@ -282,3 +288,14 @@ async function createRenderFunction(
 
   return lambda;
 }
+
+const configExts = ['.js', '.cjs', '.mjs'];
+
+function findConfig(dir: string, basename: string): string | undefined {
```

Comment:
Let's move this to a different file so that the logic can be shared with prepare-cache